discord_slash/client.py

```diff
@@ -847,6 +880,203 @@ def wrapper(cmd):
 
         return wrapper
 
+    def add_component_callback(
+        self,
+        callback: typing.Coroutine,
+        *,
+        messages: typing.Union[int, discord.Message, list] = None,
+        components: typing.Union[str, dict, list] = None,
+        use_callback_name=True,
+        component_type: int = None,
+    ):
+        """
+        Adds a coroutine callback to a component.
+        Callback can be made to only accept component interactions from a specific messages
+        and/or custom_ids of components.
+
+        :param callback: The coroutine to be called when the component is interacted with. Must accept a single argument with the type :class:`.context.ComponentContext`.
+        :type callback: Coroutine
```

Comment:
```suggestion
        :param Coroutine callback: The coroutine to be called when the component is interacted with. Must accept a single argument with the type :class:`.context.ComponentContext`.
```
These can be simplified to be on one line.